src/test/common/steps/complete-flow.js

```diff
@@ -1,6 +1,6 @@
-const { Given, When } = require('cucumber')
+const { Given, When, Then } = require('cucumber')
 
-const { selectYesOnPregnancyPage, enterNameAndSubmit, enterNinoAndSubmit, enterDateOfBirth } = require('./common-steps')
+const { selectYesOnPregnancyPage, enterNameAndSubmit, enterNinoAndSubmit, enterDateOfBirth, enterCardAddress } = require('./common-steps')
```

Comment:
Not critical as just formatting preference, but I would suggest knocking these onto seperate lines:
```
{
  selectYesOnPregnancyPage,
  enterNameAndSubmit, 
  ...
} = require('./common-steps')
```

Reply:
Done.